examples/childchain-exit.js

```diff
@@ -0,0 +1,144 @@
+
+let bobSpentOnGas
+
+async function exitChildChain () {
+  let aliceRootchainBalance = await web3.eth.getBalance(aliceAddress)
+  let bobRootchainBalance = await web3.eth.getBalance(bobAddress)
+
+  let aliceChildchainBalanceArray = await childChain.getBalance(aliceAddress)
+  let bobChildchainBalanceArray = await childChain.getBalance(bobAddress)
+
+  console.log(`Alice's rootchain balance: ${aliceRootchainBalance}`)
+  console.log(`Bob's rootchain balance: ${bobRootchainBalance}`)
+
+  console.log(`Alice's childchain balance: ${aliceChildchainBalanceArray.length === 0 ? 0 : aliceChildchainBalanceArray[0].amount}`)
+  console.log(`Bob's childchain balance: ${bobChildchainBalanceArray.length === 0 ? 0 : bobChildchainBalanceArray[0].amount}`)
+
+  // get deposit UTXO and exit data
+  let aliceUtxos = await childChain.getUtxos(aliceAddress)
+  let aliceUtxoToExit = aliceUtxos[0]
+
+  let bobUtxos = await childChain.getUtxos(bobAddress)
+  let bobUtxoToExit = bobUtxos[0]
+
+  console.log(`Alice's UTXOS = ${JSON.stringify(aliceUtxoToExit, undefined, 2)}`)
+  console.log(`Bob's UTXOs = ${JSON.stringify(bobUtxoToExit, undefined, 2)}`)
+
+  const exitData = await childChain.getExitData(bobUtxoToExit)
+
+  // start a standard exit
+  const startExitReceipt = await rootChain.startStandardExit(
+    exitData.utxo_pos,
+    exitData.txbytes,
+    exitData.proof,
+    { privateKey: bobPrivateKey, from: bobAddress }
+  )
+
+  console.log(`Started standard childchain exit. Start standard exit receipt: ${JSON.stringify(startExitReceipt, undefined, 2)}`)
+
+  const startExitTransaction = await web3.eth.getTransaction(startExitReceipt.transactionHash)
+
+  // track gas spendings
+  bobSpentOnGas = web3.utils.toBN(startExitTransaction.gasPrice).muln(startExitReceipt.gasUsed)
+
+  console.log(`Gas used for standard childchain exit: ${bobSpentOnGas}`)
+
+  // wait for challenge period to be over
+  let waitMs =
+    await rootChain.plasmaContract.methods.getExitableTimestamp(exitData.utxo_pos.toString()).call()
+  waitMs = (Number(waitMs) - Math.trunc(Date.now() / 1000)) * 1000
+
+  console.log(`Waiting for ${waitMs} millis for challenge period to end...`)
+
+  await wait(waitMs)
+
+  console.log(`Challenge period to finished`)
+
+  // call processExits() after challenge period is over
+  const processExitsPostChallengeReceipt = await rootChain.processExits(
+    transaction.ETH_CURRENCY, 0, 1,
+    { privateKey: bobPrivateKey, from: bobAddress }
+  )
+
+  console.log(`Started post-challenge childchain process exits. ` +
+      `Post-challenge childchain process exits receipt: ${JSON.stringify(processExitsPostChallengeReceipt, undefined, 2)}`)
+
+  const processExitsPostChallengeTxn =
+      await web3.eth.getTransaction(processExitsPostChallengeReceipt.transactionHash)
+
+  console.log(`Gas used for childchain process exits: ${web3.utils.toBN(processExitsPostChallengeTxn.gasPrice).muln(processExitsPostChallengeReceipt.gasUsed)}`)
```

Comment:
this is used during integration test specifically for testing purpose (we wanted to have high guarantees around how much gas we are using)

I don't think the users will need to know this in our code examples :)

Reply:
Ok, will remove it.

Reply:
I'll add a comment to the README regarding gas costs, and balances not being precisely as expected due to gas costs. For newcomers, if the math does work out, this might catch them off guard.